src/cascadia/TerminalApp/ColorPickupFlyout.xaml

```diff
@@ -2,106 +2,158 @@
     x:Class="TerminalApp.ColorPickupFlyout"

```

Comment:
(revert this one too)

Reply:
I think it's something to do with clang format, but I don't know how to fix it.